datatypes/src/commonMain/kotlin/at/asitplus/crypto/datatypes/misc/PointCompression.kt

```diff
@@ -29,7 +29,7 @@ fun Signum.toUInt(): UInt = if (this) 1U else 0U
  * the compression bit is defined as 2 + (y mod 2) for all curves
  * We assume y is big-endian encoding of valid y coordinate!
  */
-fun CryptoPublicKey.EC.compressY(): Signum = (y.last() and 1.toByte() == 1.toByte())
+fun CryptoPublicKey.EC.compressedY(): Signum = y.residue.bitAt(0)
```

Comment:
It is clearly a function, so why past tense?

Reply:
actually thinking about it, should just be a getter